src/Uno.Extensions.Maui.UI/Extensibility/IMauiEmbeddingExtension.cs

```diff
@@ -18,22 +18,22 @@ public interface IMauiEmbeddingExtension
 	/// Initializes the Maui embedding extension with the specified application.  
```

Comment:
Can we add tests with these changes?

Reply:
I'm not sure how to best test this, we may need to track this in a new issue as we need this in for the release